tests/core/core-dump/task.yaml

```
@@ -1,5 +1,9 @@
 summary: Make sure we can generate core dumps on UC
 
+details: |
+  Checks it is possible to generate core dumps on UC. Make sure that dumps
```

```suggestion
  Check it is possible to generate core dumps on UC. Make sure that dumps
```

We should pick either "Checks ..." or "Check.... " and stick to the form. I'm picking "check" since that was used in earlier patches.